DOC-636: Final set of 5.5 release notes (#1714)
tinymce/tinymce-docs: release-notes/release-notes55.md

- To view the CSS files, see: [tinymce-dist GitHub Repository - Content CSS files](https://github.com/tinymce/tinymce-dist/tree/master/skins/content).

### New listbox component for custom dialogs

The new listbox component for custom dialogs is a composite component with a label and a dropdown list of options for users to select from. This component displays a menu-like dropdown and allows nested options to be shown to users.

For information on the listbox dialog component, see: [Dialog components - listbox]({{site.baseurl}}/ui-components/dialogcomponents/#listbox).

### New `hasPlugin` for detecting the availability of plugins

The new `hasPlugin` API method can be used to detect if a plugin is present in the editor configuration or if the plugin has been loaded. This API is useful if custom functionality such as a custom plugin, is dependent on the APIs of a plugin.

For information on the `editor.hasPlugin()` API, see: [tinymce.Editor APIs - `hasPlugin`]({{site.baseurl}}/api/tinymce/tinymce.editor/#hasplugin).

### New StyleSheetLoader `unload` and `unloadAll` APIs

> Two new APIs have been for unloading CSS files:

Reviewer: ```suggestion
Two new APIs have been added for unloading CSS files:
```

- `unload` - Unloads the specified CSS file if it is not in use.

> - `unloadAll` - Unloads all CSS files not in use.

Reviewer 2: This is incorrect. It takes a list of URLs unloads the ones not in use (basically the same as `unload`, except it takes multiple URLs).

Author: ```suggestion
- `unloadAll` - Accepts a list of URLs and unloads any CSS files not in use.
```
@lnewson ?

Reviewer 2: LGTM :+1:

- The `editor.selection.setContent()` API, see: [tinymce.dom.Selection APIs - `setContent`]({{site.baseurl}}/api/tinymce.dom/tinymce.dom.selection/#setcontent).

### Changed the Image Tools context toolbar icon

The icon for the Image Tools _image options_ toolbar button has changed from an ellipse icon (![Icon consisting of three dots horizontally aligned across the center of the icon]({{site.baseurl}}/images/icons/image-options.svg)) to match the image icon (![Icon consisting of two hills, a sun above and a border representing a picture frame]({{site.baseurl}}/images/icons/image.svg)).

For information on:

- The Image Tools plugin, see: [Image Tools plugin]({{site.baseurl}}/plugins/imagetools/).
- The Image Tools context toolbar, see: [Image Tools plugin - `imagetools_toolbar`]({{site.baseurl}}/plugins/imagetools/#imagetools_toolbar).

For a list of available:

- Image Tools toolbar buttons, see: [Image Tools plugin - Toolbar buttons]({{site.baseurl}}/imagetools/#toolbarbuttons).
- {{site.productname}} icons, see: [Icons Available for TinyMCE]({{site.baseurl}}/advanced/editor-icon-identifiers/).

> ### Improved navigation and deleting for media elements without the `media` plugin

Reviewer 2: ```suggestion
### Improved navigation and deletion for media elements without the `media` plugin
```
Maybe?
